scripts/zkai_d64_rmsnorm_swiglu_statement_fixture.py

```
@@ -0,0 +1,518 @@
+    return statement_payload(reference, binding)
+
+
+def _statement_commitment_from_payload(statement: dict[str, Any]) -> str:
+    payload = copy.deepcopy(statement)
+    payload.pop("statement_commitment")
+    return blake2b_commitment(payload, "ptvm:zkai:d64-statement:v1")
+
+
+def validate_statement(statement: dict[str, Any], expected: dict[str, Any] | None = None) -> None:
+    if not isinstance(statement, dict):
+        raise StatementFixtureError("statement must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)